pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -24,6 +24,10 @@ func AddTablesMigrations(mg *migrator.Migrator) {
 	mg.AddMigration("add last_applied column to alert_configuration_history", migrator.NewAddColumnMigration(migrator.Table{Name: "alert_configuration_history"}, &migrator.Column{
 		Name: "last_applied", Type: migrator.DB_Int, Nullable: false, Default: "0",
 	}))
+
+	mg.AddMigration("add index in alert_image on url", migrator.NewAddIndexMigration(migrator.Table{Name: "alert_image"}, &migrator.Index{
```

Comment:
We have migration functions dedicated to some parts of the code. We should move that to `addAlertImageMigrations`.

Reply:
We discussed this weekly and decided to move forward with a linear history instead of continuing with grouping migrations.